### PR TITLE
Add recipe for estscan version 3.0

### PR DESCRIPTION
**Pull request by gregvonkuster**
* [x] I have read the guidelines above.
* [x] This PR adds a new recipe.
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/estscan/3.0/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  url: https://sourceforge.net/projects/estscan/files/estscan/3.0/estscan-3.0.tar.gz
+  md5: 1252c862b41b03b59687f5ba148b77cd
+
+build:
+  number: 0
+  skip: True #[osx]
+
+requirements:
+  build:
+    - libgfortran
```
Comment (bgruening) [inline recipes/estscan/3.0/meta.yaml]: Have you tried without libgfortran. I think this is not needed if you use GCC.

**Comment by gregvonkuster** [comment]
@bgruening I've commented out fortran components compiled with F77 since this recipe does not use them.  This has allowed me to eliminate the fortran requirement,

**Comment by bgruening** [comment]
LGTM!

**Comment by gregvonkuster** [comment]
Thanks @bgruening ;)

**Pull request merged**